doc.php

```diff
@@ -35,8 +35,20 @@
   }
   if (!isset($repository))
     $repository = 'omichel';
+
+  # get HEAD commit SHA, to ensure that when master is updated the latest version is cached by the CDN
```

Comment:
All this sequence is only used when `$branch === ''`, it would seem more logical to me to move it in the test below.